babel.config.js

```diff
@@ -1,7 +1,12 @@
 module.exports = function(api) {
   api.cache(true)
   return {
-    presets: [['module:metro-react-native-babel-preset'], ['react-app']],
+    presets: [['module:metro-react-native-babel-preset'], ['react-app'], ['@babel/preset-env', {
```

Comment:
looks fine but this will work for master branch only
in react-native those files are unused. 

for native app you should configure plugin in metro.config, for web - config-overrides.js

please:
- create separate dev branch e.g. 1545-optimize_lodash_imports-native from react-native
- configure lodash plugin by changing corresponding plugins
- adjust lodash imports (by merging them from this branch)
- create separate PR linked to this task

Reply:
Done. new PR here - https://github.com/GoodDollar/GoodDAPP/pull/1566